pulseguardian/management.py

```diff
@@ -9,106 +9,106 @@
 
 import requests
 
+from pulseguardian import config
 MAX_RETRY = 5
 
 
 class PulseManagementException(Exception):
     pass
 
+"""
+Wrapper around the RabbitMQ management plugin's REST API.
+"""
 
-class PulseManagementAPI(object):
-    """Wrapper around the RabbitMQ management plugin's REST API.
-
-    :param url: Management API URL.
-    :param management_port: Port used by the management plugin.
-    :param user: RabbitMQ user with administrator privilege.
-    :param password: Password of the RabbitMQ user.
-    """
-    exception = PulseManagementException
-
-    def __init__(self, management_url, user, password):
-        self.management_url = management_url.rstrip('/') + '/'
-        self.management_user = user
-        self.management_password = password
-
-    def _api_request(self, path, method='GET', data=None):
-        session = requests.Session()
-        url = '{0}{1}'.format(self.management_url, path)
-        request = requests.Request(method, url,
-                                   auth=(self.management_user,
-                                         self.management_password),
-                                   data=json.dumps(data)).prepare()
-        request.headers['Content-type'] = 'application/json'
-        response = None
-
-        for i in xrange(MAX_RETRY):
-            try:
-                response = session.send(request)
-                break
-            except (requests.ConnectionError, socket.error):
-                logging.exception('Failed to connect to the RabbitMQ server.')
-
-        if response is None or not response.content:
-            return None
 
+def _api_request(path, method='GET', data=None):
+    session = requests.Session()
+    management_url = config.rabbit_management_url
```

Comment:
Seems unnecessary to alias config.rabbit_management_url, particularly since it is only used once, on the following line.